packages/nimble-components/src/checkbox/index.ts

```diff
@@ -16,7 +17,23 @@ declare global {
 /**
  * A nimble-styled checkbox control.
  */
-export class Checkbox extends FoundationCheckbox {}
+export class Checkbox extends FoundationCheckbox {
+    /**
+     * @public
+     * @remarks
+     * HTML Attribute: tabindex
+     */
+    @attr({ attribute: 'tabindex', converter: nullableNumberConverter })
+    public override tabIndex!: number;
+
+    /**
+     * @internal
+     */
+    public get resolvedTabIndex(): string | undefined {
+        const tabIndex = this.tabIndex ?? 0;
+        return this.disabled ? undefined : tabIndex.toString();
```

Comment:
nit / pref, using in a template for the explicit conversion is a little more compact
```suggestion
        return this.disabled ? undefined : `${tabIndex}`;
```